python/lsst/ip/isr/overscan.py

```diff
@@ -547,27 +576,82 @@ def maskOutliers(self, imageArray):
         axisMedians = median
         axisStdev = 0.74*(uq - lq)  # robust stdev
 
+        # Replace pixels that have excessively large stdev values
+        # with the median of stdev values.  A large stdev likely
+        # indicates a bleed is spilling into the overscan.
+        axisStdev = np.where(axisStdev > 2.0 * np.median(axisStdev),
```

Comment:
It can take almost any value and do a similar thing.  It was added for the case below:
```
overscan = [[0.0 0.0 0.0 0.0 0.0] # Furthest row from imaging section.
            [0.0 0.0 100000.0 0.0 0.0]
            [0.0 0.0 100000.0 0.0 0.0]] # Closest row to the imaging section.
axisMedian = [0.0 0.0 100000.0 0.0 0.0]
axisStdev = [0.0 0.0 37000.0 0.0 0.0]
```
With these values, the "bleed" column rejects nothing, as all pixels are within 3 sigma, so some sigma clipping of the STDEV values is required.  I chose 2.0 because in all the plots I looked at, the real STDEV values were tightly clustered about the median, so the penalty for over clipping makes little difference to the masking step.